lib/src/protocol/http/mod.rs

```diff
@@ -614,9 +617,22 @@ impl<Front: SocketHandler> Http<Front> {
         }
 
         let proto = self.protocol_string();
+        let tags = host.inner.and_then(|host| {
+            let hostname = match host.split_once(':') {
+                None => host,
+                Some((hn, _)) => hn,
+            };
+
+            self.listener.borrow().get_tags(hostname).map(|tags| {
+                tags.iter()
+                    .map(|(k, v)| format!("\"{}={}\"", k, v))
+                    .collect::<Vec<_>>()
+                    .join(" ")
+            })
+        });
 
         info_access!(
-            "{}{} -> {}\t{} {} {} {}\t{} {} {}\t{}",
+            "{}{} -> {}\t{} {} {} {}\t{} {} {}\t{} {}",
```

Comment:
I'm still not sure that adding tags after a user controlled input is a good idea since it might lead to harder parsing later.

Reply:
Where do you prefer to put the string as it is an user input as well..

Reply:
Tags are defined by an administrator or an admin tool, not by a random user, while backends may return status lines operated by random users. I believe that parsing data after untrusted input makes parsing harder since you have to detect when the status line ends to start parsing your tags.

You could put it after the host maybe?

Reply:
I am convinced by @BlackYoup 's argument on trusted vs. non-trusted input in log lines. After the host looks okay to me, as a place to write logs at.